pkg/commands/run.go

```diff
@@ -489,6 +494,31 @@ func (e *Executor) createPrinter(format string, w io.Writer) (printers.Printer,
 	return p, nil
 }
 
+func (e *Executor) printStats(issues []result.Issue) {
```

Comment:
Do we need to add a unit or integration test?

Reply:
no need for now, we will wait for users' feedback before doing something because the current implementation is simple and it can evolve with feedback.